src/py/mat3ra/notebooks_utils/ipython/ui.py

```
@@ -0,0 +1,198 @@
+    Args:
+        obj (dict): Object to display as nicely-formatted JSON
+        interactive_viewer (bool): Whether to use the interactive viewer or not
+        level (int): The level to which the JSON should be expanded by default
+    """
+    if interactive_viewer:
+        if isinstance(obj, (dict, list)):
+            json_str = json.dumps(obj)
+        else:
+            json_str = obj
+
+        id = str(uuid.uuid4())
```

Ruff (0.15.12)

[error] Variable `id` is shadowing a Python builtin

(A001)